### ISSUE TITLE
Unable to stop named app

### ISSUE THREAD

**Issue by llamafilm** [docker/for-mac#5557]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics

I have waited over 20 minutes on the troubleshooting page but it just says "Diagnosing..."  I also tried relaunching Docker but I think something is wrong with the diagnostics.

### Expected behavior
I want to stop a named "app" (project) started by docker-compose.

### Actual behavior
Nothing happens

### Information
Without a project name it works fine.  When I launch with a project name like `docker-compose -p test1 up -d`, I can stop each individual container by clicking its stop button in Docker Desktop.  But Clicking the stop button for the whole app does nothing.  I see this in the logs, but no follow-up:

```
2021-04-08 15:54:35.688280-0700  localhost com.docker.backend[4824]: received new cli usage: {Command:compose stop Context:moby Status:success Source:docker-compose}
2021-04-08 15:54:35.689439-0700  localhost com.docker.backend[4824]: cli: POST /usage 200 "python-requests/2.24.0" ""
```

  - macOS Version: 10.15.7
  - Docker Desktop Version: 3.2.2

### Steps to reproduce the behavior
Using this compose file, run `docker-compose -p immortal up -d`

```
version: '3'
services:
  db:
    image: mysql
    environment :
      MYSQL_ALLOW_EMPTY_PASSWORD: "true"
  web:
    image: php:apache
```

**Comment by stephen-turner** [comment]
Thanks for the report. I believe this is the same as docker/for-win#10564, and is fixed in version 3.3.0.

**Comment by llamafilm** [comment]
Yes, it is fixed in 3.3.0, thanks!
